spec/poros/food_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+RSpec.describe Food do
+  it "exists" do
+    total_hits = 9001
+    attrs = {
+      gtinUpc: "041497083411",
+      lowercaseDescription: "cheddar cheese",
+      brandOwner: "Weis Markets, Inc.",
+      ingredients: "PASTEURIZED MILK, CHEESE CULTURE, SALT, ENZYMES AND ANNATTO [ VEGETABLE COLOR]. POTATO STARCH, CORN STARCH AND CALCIUM SULFATE ADDED TO PREVENT CAKING. NATAMYCIN (A NATURAL MOLD INHIBITOR)."
+    }
+
+    food = Food.new(total_hits, attrs)
```

Comment:
Great job passing both the total_hits and attrs here so that you expect to see both with a single call.